src/sbmlutils/report/formating.py

```diff
@@ -79,14 +108,24 @@ def equationStringFromReaction(
     return " ".join([left, sep, right])
 
 
-def _modifierEquation(modifierList):
+def _modifierEquation(modifierList: List[libsbml.ModifierSpeciesReference]) -> str:
```

Comment:
This is an libsbml object, i.e. `libsbml.ListOfSpeciesReferences`
 These start with `libsbml.ListOf*`
You have to change the `idea.properties` file of pycharm with
```
idea.max.intellisense.filesize=12500
```
to get code completion and help for the libsbml functionality. You can then browse the objects via autocomplete in pycharm.
See below 
![image](https://user-images.githubusercontent.com/900538/107696221-faf05e00-6cb1-11eb-856e-608a4ddc2f55.png)